templates/404.njk

```diff
@@ -2,6 +2,7 @@
 
 {% set thispage = '404' %}
 {% set pagetitle = pagetitle if pagetitle else '404' %}
+{% set pagedesc = 'Errr' %}
```

Comment:
Errr? Not Error? ;-;

Reply:
Whom are we sharing this link

Reply:
IDK fam. In case you misspell the link or something, you want them to see Errr instead of error? ;-;